imports/plugins/included/product-detail-simple/client/components/productDetail.js

```diff
@@ -70,7 +92,122 @@ class ProductDetail extends Component {
         </Toolbar>
       );
     }
+    return null;
+  }
+  switchDigital(e) {
+    this.setState({digital: e.target.checked});
+    const productId = this.props.product._id;
+    const checked = e.target.checked;
```

Comment:
You used `e.target.checked` three times in this block. Why not assign it to a variable and pass it around?